rcon/rcon.py

```diff
@@ -732,7 +732,61 @@ def get_gamestate(self) -> GameStateType:
             Rcon.get_team_objective_scores,
             Rcon.get_round_time_remaining,
         ):
-            return super().get_gamestate()
+            gamestate = super().get_gamestate()
```

Comment:
This does not handle situations with multiple occurrences of the same map in the sequence resulting into wrongful information. I think it is better to say we don't really know what the next map oppose to providing a wrong next map information.